src/macros.rs

```diff
@@ -470,19 +470,10 @@ macro_rules! crate_authors {
         impl Deref for CargoAuthors {
             type Target = str;
 
-            #[allow(unsafe_code)]
             fn deref(&self) -> &'static str {
-                static ONCE: Once = Once::new();
-                static mut VALUE: *const String = 0 as *const String;
-
-                unsafe {
-                    ONCE.call_once(|| {
-                        let s = env!("CARGO_PKG_AUTHORS").replace(':', $sep);
-                        VALUE = Box::into_raw(Box::new(s));
-                    });
-
-                    &(*VALUE)[..]
-                }
+                let s: Box<String> = Box::new(env!("CARGO_PKG_AUTHORS").replace(':', $sep));
+                let s2 = Box::leak(s);
```

Comment:
This leaks a fresh `Box<String>` on every single `Deref` invocation.

Reply:
Hang on, is this macro invoked at runtime or at compile time? If it's at runtime, then this entire approach is probably a bad idea and we need to move this logic to build.rs instead.

Reply:
Fixed, thanks!